repos/spack_repo/builtin/packages/eigen/package.py

```diff
@@ -23,12 +23,16 @@ class Eigen(CMakePackage, ROCmPackage):
     license("MPL-2.0")
 
     version("master", branch="master")
-    version("3.4.0-44-ge7248b26a", commit="e7248b26a1ed53fa030c5c459f7ea095dfd276ac")
     version(
-        "3.4.0",
-        sha256="8586084f71f9bde545ee7fa6d00288b264a2b7ac3607b974e54d13e7162c1c72",
+        "5.0.0",
+        sha256="315c881e19e17542a7d428c5aa37d113c89b9500d350c433797b730cd449c056",
         preferred=True,
```

Comment:
`preferred` is not necessary unless you want to change the ordering. In this case, that's not the case (anymore).
```suggestion
```